### ISSUE TITLE
Stale docker containers and dockerhub does not link to git Dockerfiles

### ISSUE THREAD

**Issue by kbroughton** [panther-labs/panther#369]
### Describe the bug
mage deploy fails with
04:28:57	FATAL	.setup/swagger not found (stat .setup/swagger: no such file or directory): run 'mage setup:all'

This error was fixed in master 19 days ago, but the container has not been rebuilt.
dev.sh runs the panther-development-pack container.
Visiting https://hub.docker.com/r/pantherlabs/panther-development-pack
we can see that the latest build was 1 month ago. Worse, the Dockerfiles are not linked.
panther-development-pack is FROM panther-buildpack
which appears to relate to build/ci/Dockerfile, but neither of these contain the source code.

### Steps to reproduce

1. git clone latest-panther
2. ./dev.sh
3. mage deploy
4. see the error

### Expected behavior
The Dockerfiles are located in
./build/ci/Dockerfile
./build/deployment/Dockerfile
./deployments/web/Dockerfile

It's not obvious which Dockerfile maps to which hub.docker.com image.

https://hub.docker.com/r/pantherlabs/panther-buildpack should link to the Dockerfile on github.

### Environment

OSx
- Panther version or commit: 462d1372918d48c16c64d0aa5af90933dca34f94
- OS: [e.g. MacOS 10.16.3]
- Browser: [Chrome]

**Comment by austinbyers** [comment]
@3nvi is the Panther deployment docker image up to date now?

**Comment by 3nvi** [comment]
> It's not obvious which Dockerfile maps to which hub.docker.com image.

> https://hub.docker.com/r/pantherlabs/panther-buildpack should link to the Dockerfile on github.

You are right on that. It's something we forgot to do and I'll make sure to update it.

> git clone latest-panther
./dev.sh
mage deploy
see the error

According  to our [quick start docs](https://docs.runpanther.io/quick-start#step-3), the command to deploy is:

`mage setup:swagger deploy` and **not** just `mage deploy`. This is because swagger recently became a dependency for deployment. 

> but neither of these contain the source code.

The images themselves **never hold** any source code intentionally, since each image needs to hold the source code which might differ from one person to another. For example, you might modify the `panther_config.yml` to customize your deployment. If you did that, our image wouldn't be able to pick it up since it would always see the code of a certain release. At the same time, it wouldn't make sense to tell you to go inside the image and modify files, since those files wouldn't be in sync with your locally cloned repo. To simplify things, the public images only contain the necessary setup and the code gets mounted from your local machine. We are currently discussing whether `swagger` installation should be part of the public image or not (since it's a requirement that's unrelated to the configurable source code). I created [an issue to track that](https://github.com/panther-labs/panther/issues/603)

####

In short:

- You are right about Dockerfile linking, will update asap
- Source code is intentionally left outside of the image
- Deployment command has changed to `mage setup:swagger deploy`

@kbroughton Thanks again for reporting that. Have I addressed your concerns or have I missed something?

**Comment by 3nvi** [comment]
Update:

@kbroughton I just saw that this is an old issue when we were on alpha (thought it was related to our recent v1 deployment). Please make sure to check our [stable release](https://github.com/panther-labs/panther/releases/tag/v1.0.0) which will **always** be backwards compatible and let us know.

**Comment by 3nvi** [comment]
Closing this issue as the only actionable item (link from dockerhub to dockerfiles) has been implemented:

- https://hub.docker.com/repository/docker/pantherlabs/panther-buildpack
- https://hub.docker.com/repository/docker/pantherlabs/panther-development-pack
